pkgs/by-name/de/deskflow/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  pkg-config,
+  tomlplusplus,
+  cli11,
+  gtest,
+  libei,
+  libportal,
+  libX11,
+  libxkbfile,
+  libXtst,
+  libXinerama,
+  libXi,
+  libXrandr,
+  libxkbcommon,
+  pugixml,
+  python3,
+  gdk-pixbuf,
+  libnotify,
+  qt6,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "deskflow";
+
+  version = "1.17.0";
+
+  src = fetchFromGitHub {
+    owner = "deskflow";
+    repo = "deskflow";
+    rev = "${version}+r1";
```

Comment:
Where is this tag? There's only `v1.17.0`

Reply:
The old tag was deleted upstream

Reply:
This is not a good sign. You'd better write a commit has instead.